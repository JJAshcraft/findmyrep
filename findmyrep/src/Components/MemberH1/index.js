import styled from "styled-components";
import H1 from "../H1";

const MemberH1 = styled(H1)`
  color: #06a7ed;
  display: inline;
`;

export default MemberH1;
