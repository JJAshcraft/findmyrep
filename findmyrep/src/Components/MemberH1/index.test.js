import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import MemberH1 from "./index.js";

test("MemberH1 component renders with proper styling", () => {
  const tree = renderer.create(<MemberH1 />).toJSON();
  expect(tree).toMatchSnapshot();
});
