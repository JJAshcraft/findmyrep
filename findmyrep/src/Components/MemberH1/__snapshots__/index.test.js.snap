// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberH1 component renders with proper styling 1`] = `
.c0 {
  font-size: 20px;
  font-weight: 500;
  color: #06a7ed;
  color: #06a7ed;
  display: inline;
}

<h1
  className="c0"
/>
`;
